src/data/default.json

```diff
@@ -419,7 +420,14 @@
         "disabled": false,
         "url": "http://localhost:8113/",
         "enabled": false,
-        "subMenu": [{ "id": "load-data", "to": "#" }]
+        "subMenu": [
+          {
+            "id": "load-data",
+            "to": "#",
+            "experimental": false,
+            "icon": "pi pi-server"
```

Comment:
have different icons for different services

Reply:
Done